internal/wire/records_test.go

```diff
@@ -0,0 +1,72 @@
+package wire
+
+import (
+	"bufio"
+	"io"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+func fetchRecords(recordsPath string) ([]testCase, error) {
+
+	// Fetch resursively every file path with ".bin" extension from recordsPath directory
```

Comment:
Do we prefer to fetch files recursively from directory or not? I think that's nice because in the future we could seperate logs from different sessions into seperate directories but it's up to you.

Reply:
Sure, let’s do it.

Not sure I would use a verb “fetch”, though. Read? Load?